src/client/src/consumer.rs

```diff
@@ -392,7 +405,13 @@ impl ConsumerConfig {
 
 /// The individual record for a given stream.
 pub struct Record {
```

Comment:
these are already in the batch header why creating duplicate information?

Reply:
I removed the `partition_index` and `log_start_offset` as they ended up not being needed by the CLI consumer